content/systems/filecoin_files/piece/_index.md

```diff
@@ -8,20 +8,36 @@ dashboardAudit: 0
 dashboardTests: 0
 ---
 
-# Piece - Part of a file
+# The Filecoin Piece & Data Representation in Filecoin
 ---
 
+The _Filecoin Piece_ is the main _unit of negotiation_ for data that users store on the Filecoin network. The Filecoin Piece is _not a unit of storage_, it is not of a specific size, but is upper-bounded by the size of the _Sector_. A Filecoin Piece can be of any size, but if a Piece is larger than the size of a Sector that the miner supports it has to be split into more Pieces so that each Piece fits into a Sector.
+
 A `Piece` is an object that represents a whole or part of a `File`,
 and is used by `Clients` and `Miners` in `Deals`. `Clients` hire `Miners`
 to store `Pieces`. 
 
-The piece data structure is designed for proving storage of arbitrary
+The Piece data structure is designed for proving storage of arbitrary
 IPLD graphs and client data. This diagram shows the detailed composition
-of a piece and its proving tree, including both full and bandwidth-optimized
-piece data structures.
+of a Piece and its proving tree, including both full and bandwidth-optimized
+Piece data structures.
 
 
 {{< figure src="pieces.png" title="Pieces, Proving Trees, and Piece Data Structures" zoom="true">}}
 
+## Data Representation in the Filecoin Network
+
+It is important to highlight that data submitted to the Filecoin network go through several transformations before they come to the format at which the `StorageProvider` stores it.
+
+1. When a piece of data, or file is submitted to Filecoin (in some raw system format) it is transformed into a _UnixFS DAG style data representation_ (in case it is not in this format already, e.g., from IPFS-based applications). The hash that represents the root of the IPLD DAG of the UnixFS file is the _Payload CID_, which is used in the Retrieval Market. The Payload CID is identical to an IPFS CID.
+2. In order to make a _Filecoin Piece_ the UnixFS IPLD DAG is serialised into a ["Content-Addressable aRchive" (.car)](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md#summary) file, which is in raw bytes format.
+3. The resulting .car file is _padded_ with extra bits.
```

Comment:
Perhaps it's best to leave this vague and simple, but if it helps I tried to outline this (mainly in the context of how we get to CIDs and what those CIDs indicate) the other day @ https://github.com/ipfs-shipyard/ipld-explorer/issues/64#issuecomment-663401567

The main take-away for padding is that we need to get to a ^2 size so we can do a clean binary merkle. Before getting to that ^2, an extra 2 zero bits per 254 bits need to be added. So we take the original input, add zero bytes to the end of it until we reach exactly the size that we'll need in order to pad it out with those 2-zeros per 254 bits and end up with a ^2. Kind of tricky to explain when you have to work backward like that. Something like: zero-pad up to the `254/256` of the next ^2 value, then do the 2-bits-per-254 on top of that.